tests/core/filter/manual/bordershorthandtransformation.md

```diff
@@ -0,0 +1,13 @@
+@bender-tags: bug, 1010, 4.8.1, bug
+@bender-ui: collapsed
+@bender-ckeditor-plugins: wysiwygarea, toolbar, sourcearea, table, tabletools, contextmenu, dialogadvtab
+
+----
+
+1. Copy-paste table to editor.
+2. Switch between sourcemode and wysiwyg.
+3. Check TD element's style
```

Comment:
This scenario is a little confusing IMHO. The first step says to copy/paste table to editor, but there is no table, there is only table HTML.
Maybe it should say something like _Copy table HTML. Paste into editor in source mode._ and then the rest of scenario with switching and checking color. WDYT?